File: app/src/main/java/com/fintech/superadmin/data/browseplan/Plan.java
package com.fintech.superadmin.data.browseplan;

import com.google.gson.annotations.SerializedName;

public class Plan {
    @SerializedName("desc")
    public String desc;
    @SerializedName("last_update")
    public String lastUpdate;
    @SerializedName("rs")
    public String rs;
    @SerializedName("validity")
    public String validity;
}

File: app/src/main/java/com/fintech/superadmin/data/deer_response/Plan.kt
package com.fintech.superadmin.data.deer_response

import com.google.gson.annotations.SerializedName

data class Plan(
    @SerializedName("desc")
    val desc: String,
    @SerializedName("last_update")
    val lastUpdate: String,
    @SerializedName("rs")
    val rs: String,
    @SerializedName("validity")
    val validity: String
)
